tasks/celery.yml

```diff
@@ -6,11 +6,22 @@
             group=root
             mode=0600
   with_items:
-    - { name: "default", command: "worker", flags: "{{ celery_worker_extra_args|default('--loglevel=INFO') }}" }
+    - { name: "default", command: "worker", flags: "{{ celery_worker_extra_args|default('--loglevel=INFO') }}{% if celery_events %} --events{% endif %}" }
     - { name: "beat", command: "beat", flags: "--schedule={{ root_dir }}/celerybeat-schedule.db --pidfile=/var/run/celery/celerybeat.pid --loglevel=INFO" }
 
+- name: configure celery events
+  template: src=celery.conf
+            dest=/etc/supervisor/conf.d/{{ project_name }}-celery-events.conf
+            owner=root
+            group=root
+            mode=0600
+  when: celery_events
+  with_items:
```

Comment:
Let's go ahead and make this a `vars:` declaration instead, with the individual variables broken out, something like

    vars:
        name: "events"
        command: "events"
        ...